### PR TITLE
fix: add missing fields to execution outcome and test

### PR DESCRIPTION
**Pull request by austinabell**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/near/near-api-js/blob/master/CONTRIBUTING.md).
- [x] Commit messages follow the [conventional commits](https://www.conventionalcommits.org/) spec
- [x] **If this is a code change**: I have written unit tests.
- [x] **If this changes code in a published package**: I have run `pnpm changeset` to create a `changeset` JSON document appropriate for this change.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

This RPC type is missing fields that are required for proof verification. There is another `metadata` field and `compute_usage` fields that are missing also, but I opted to keeping these changes light.

I defaulted to having this be a patch changeset, but can change to minor if intended. Unclear how semver is with TS adding fields to interfaces (esp in this case since those types shouldn't be constructed manually)

## Test Plan

updated an existing test

### CODE REVIEW

File: packages/near-api-js/test/providers.test.js [hunk 1]
```diff
@@ -82,6 +82,13 @@ test('txStatusReciept with string hash and buffer hash', withProvider(async(prov
 
     const responseWithString = await provider.txStatusReceipts(outcome.transaction.hash, sender.accountId);
     const responseWithUint8Array = await provider.txStatusReceipts(base58.decode(outcome.transaction.hash), sender.accountId);
+    expect('transaction_outcome' in responseWithString).toBeTruthy();
+    expect('logs' in responseWithString.transaction_outcome.outcome).toBeTruthy();
+    expect('receipt_ids' in responseWithString.transaction_outcome.outcome).toBeTruthy();
+    expect('gas_burnt' in responseWithString.transaction_outcome.outcome).toBeTruthy();
+    expect('tokens_burnt' in responseWithString.transaction_outcome.outcome).toBeTruthy();
+    expect('executor_id' in responseWithString.transaction_outcome.outcome).toBeTruthy();
+    expect('status' in responseWithString.transaction_outcome.outcome).toBeTruthy();
```
Comment (andy-haynes) [inline packages/near-api-js/test/providers.test.js:91]: Please move  this to `packages/accounts/test/providers.test.js`

File: .changeset/rotten-monkeys-tie.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"near-api-js": patch
```
Comment (andy-haynes) [inline .changeset/rotten-monkeys-tie.md:2]: Only the changed package should be indicated here. `changeset` will bump versions of packages with a dependency on the changed packages at publish time.

**Review by andy-haynes — CHANGES_REQUESTED** [review]

File: .changeset/rotten-monkeys-tie.md (hunk 2 above)
Comment (austinabell) [reply .changeset/rotten-monkeys-tie.md:2]: oops, I must have misclicked

**Review by andy-haynes — APPROVED** [review]

**Comment by andy-haynes** [comment]
Thanks @austinabell !

**Pull request merged**